package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "service-template-node",
-  "version": "0.4.1",
+  "version": "0.5.0",
```

Comment:
Could you not bump the version here? I prefer to have it as a separate PR where `package.json` is refreshed as a whole (deps, deploy, etc)